src/clj/game/cards/agendas.clj

```diff
@@ -515,11 +515,12 @@
 
    "Personality Profiles"
    (let [pp {:req (req (pos? (count (:hand runner))))
-             :msg "force the Runner to trash 1 card from their Grip at random"
-             :effect (effect (trash (first (shuffle (:hand runner)))))}]
+             :effect (effect (trash (first (shuffle (:hand runner)))))
+             :msg (msg "force the Runner to trash 1 card from their Grip at random. Trashes: "
+                       (:title (first (:discard runner))))}]
```

Comment:
Maybe better to phrase it like "force the Runner to trash <named card> from their Grip at random."